### ISSUE TITLE
ComboBox does not support aria-label for roles with aria-role="listbox"

### ISSUE THREAD

**Issue by mraveesha** [microsoft/fluentui#23989]
### Describe the issue:

ComboBox as listbox form does not have aria-labels. Individual combo box components are able to add aria-label, but we need an aria-label for the entire list.
When we add an aria-label, the label exists only for the selected option that is selected from the dropdown. We need a label for the entire list when the dropdown is selected and all options are shown.

Screenshot below shows the list we want an aria-label for:

![image](https://user-images.githubusercontent.com/80782893/179821288-9ca2b36e-78ca-4746-a959-526f0421cf37.png)
![image](https://user-images.githubusercontent.com/80782893/179821345-0b533ea3-54cd-4468-b4bc-d55601dd8af9.png)

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

**Comment by gouttierre** [comment]
hanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [Combobox ](https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/combobox) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
